### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by Zales0123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,26 +3,111 @@
...
+| sylius/sylius | ^1.10.0 |
+
+### Overview
+
+This plugin adds integration with [Wordline Saferpay](https://www.saferpay.com) to [Sylius](https://github.com/Sylius/Sylius).
+
+#### About Saferpay
+
+Saferpay is a modern, all-in-one payment solution, particularly popular in the DACH region due to origins of its
+previous owner - SIX Payments - which is a Swiss fintech acquired by French Wordline group. As part of Wordline group,
```
Comment (zamaskowany-szachista) [inline README.md:35]: Please change the sentence to: a Swiss company acquired by Worldline - a French fintech group. 

And delete the beginning of next sentence: ~As part of Wordine group,~

File: README.md [hunk 2]
```diff
@@ -3,26 +3,111 @@
...
+### Requirements
+
+| Package       | Version |
+|---------------|---------|
+| PHP           | ^8.0    |
+| sylius/sylius | ^1.10.0 |
+
+### Overview
+
+This plugin adds integration with [Wordline Saferpay](https://www.saferpay.com) to [Sylius](https://github.com/Sylius/Sylius).
```
Comment (zamaskowany-szachista) [inline README.md:30]: Worldline. We're always missing the first "L" in this name. :)

File: README.md [hunk 3]
```diff
@@ -3,26 +3,112 @@
...
+1. [Cooperation](#cooperation)
+1. [License](#license)
+1. [About us](#about-us)
+
+### Requirements
+
+| Package       | Version |
+|---------------|---------|
+| PHP           | ^8.0    |
+| sylius/sylius | ^1.10.0 |
```
Comment (GSadee) [inline README.md:27]:
```suggestion
| sylius/sylius | ^1.10   |
```

File: docs/img/overview/payment-logos.png
Comment (GSadee) [inline docs/img/overview/payment-logos.png:1]: In the checkout screenshot we have the CHF currency, if we use this currency also here, we could probably have more payment methods

**Review by GSadee — APPROVED** [review]

**Pull request merged**